docs/content/Examples-Tutorials-Recipes/Refreshing-select-partitions.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Refreshing Select Partitions
+permalink: /recipes/refreshing-select-partitions
+category: Examples & Tutorials
+subCategory: Query acceleration
+menuOrder: 6
+---
+
+## Use case
+
+We have a dataset with orders and we want to aggregate data while having decent
+performance. Orders have a created time, so we can use
+[partitioning](https://cube.dev/docs/caching/using-pre-aggregations#partitioning)
+by time to optimize pre-aggregations build and refresh time. The problem is the
+order can change status after a long time. In this case we want to rebuild only
+partitions associated with this order.
+
+In the recipe below, we'll learn how to use the
+[`refreshKey`](https://cube.dev/docs/schema/reference/pre-aggregations#parameters-refresh-key-sql)
+together with the
+[`FITER_PARAMS`](https://cube.dev/docs/schema/reference/cube#filter-params) for
+partition separately.
+
+## Data schema
+
+Let's explore the `Orders` cube data that contains various information about
+orders, including number and status:
+
+| id  | number | status     | created_at          | updated_at          |
+| --- | ------ | ---------- | ------------------- | ------------------- |
+| 1   | 1      | processing | 2021-08-10 14:26:40 | 2021-08-10 14:26:40 |
+| 2   | 2      | completed  | 2021-08-20 13:21:38 | 2021-08-22 13:10:38 |
+| 3   | 3      | shipped    | 2021-09-01 10:27:38 | 2021-09-02 01:12:38 |
+| 4   | 4      | completed  | 2021-09-20 10:27:38 | 2021-09-20 10:27:38 |
+
+In our case, each order has `created_at` and `update_at` properties. The
+`update_at` property is the last order update timestamp. To create a
+pre-aggregation splitting to partitions, we need to specify the
+[`partitionGranularity` property](https://cube.dev/docs/schema/reference/pre-aggregations#partition-granularity).
+Partitions will be split monthly by the `created_at` dimension.
+
+```javascript
+preAggregations: {
+    orders: {
+      type: `rollup`,
+      external: true,
+      dimensions: [CUBE.number, CUBE.status, CUBE.createdAt, CUBE.updatedAt],
+      timeDimension: CUBE.createdAt,
+      granularity: `day`,
+      partitionGranularity: `month`,
+      refreshKey: {
+        sql: `SELECT max(updated_at) FROM public.orders`
```

Comment:
```suggestion
        sql: `SELECT max(updated_at) FROM public.orders` // check for updates of the updated_at property
```

Reply:
Another suggestion, I just want the user to know exactly what this `sql` statement does. Open for suggestions on how to improve the wording. :smile: